src/components/Pages/Admin/View.vue

```diff
@@ -34,14 +34,7 @@
                                             Unmerge groups views
                                         </button>
                                     </div>
-                                    <div class="checkbox form-inline mt-4">
-                                        <label class="checkbox-container">
-                                            Show menu bar
-                                            <input type="checkbox"
-                                                id="menu_bar_checkbox"
-                                                v-model="showMenuBar">
-                                            <span class="checkmark"></span>
-                                        </label>
+                                    <div class="checkbox form-inline">
```

Comment:
Because all the contents of this are now commented out, I'd prefer for this div to be commented out as well, so that we don't have empty useless elements on the page.